src/client/containers/NotificationBanner/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { FC } from 'react';
+import { first } from 'lodash';
+import styled from 'styled-components';
+import { TransitionGroup, CSSTransition } from 'react-transition-group';
+import { useAppSelector, useAppDispatch } from '@hooks';
+import { NotificationsActions, NotificationsSelectors } from '@store';
+import { Banner, Markdown, Icon, CloseIcon } from '@src/client/components/common';
```

Comment:
We have to still write a guideline for this:
- imports should follow this order: external imports -> internal imports using path alias-> internal imports using relative paths (all separated by new line)
- prefer directly importing from the direct path if its available. For `@src/client/components/common` do `@components/common`. See tsconfig.paths.json